pages/common/aws-route53.md

```diff
@@ -0,0 +1,24 @@
+# aws route53
+
+> CLI for AWS Route53 - Route 53 is a highly available and scalable Domain Name System (DNS) web service.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/route53/index.html>.
+
+- List all hosted zones, private and public:
+
+`aws route53 list-hosted-zones`
+
+- Show all records in a zone:
+
+`aws route53 list-resource-record-sets --hosted-zone-id {{zone_id}}`
+
+- Create a new, public zone using a request identifier to retry the operation safely:
+
+`aws route53 create-hosted-zone --name {{name}} --caller-reference {{request_identifier}}`
+
+- Delete a zone (it has to be empty from any records but the default NS ones):
```

Comment:
Can you rephrase this description a bit?

Reply:
Sure, but tell me in what direction, what's wrong with this one?

Reply:
mmm, not sure. :thinking: 
```suggestion
- Delete a zone (if the zone has non-defaults SOA and NS records the command will fail):
```
@CleanMachine1 could you please help?

Reply:
What I meant by this - if a zone has some/any type of record(s) beyond the records auto-created by the AWS Route53 (SOA and NS) when the zone was created (empty), the deletion will fail. 
I did not mean that SOA/NS should be the default.  
In other words, if a zone has additional records except SOA/NS, the deletion will fail.

Reply:
Yeah, I read the doc https://awscli.amazonaws.com/v2/documentation/api/latest/reference/route53/delete-hosted-zone.html:
> You can delete a hosted zone only if it contains only the default SOA record and NS resource record sets. If the hosted zone contains other resource record sets, you must delete them before you can delete the hosted zone. If you try to delete a hosted zone that contains other resource record sets, the request fails, and Route 53 returns a HostedZoneNotEmpty error.

Reply:
On a second thought - a user will get error trying to delete a non-empty zone anyway, may be worth removing this comment, leave just "Delete a zone" ?

Reply:
Yeah, we can't remove the clarification